src/components/spaces/show/fact_sidebar.js

```diff
@@ -0,0 +1,35 @@
+import React, {Component, PropTypes} from 'react'
```

Comment:
Is this file actually used? It looks like you don't import this new component below...